### ISSUE TITLE
my python script can't exit on linux(centos 7), but can exit on win10,all python=3.7

### ISSUE THREAD

**Issue by duo66** [pahaz/sshtunnel#211]
```python
# -*- coding:UTF-8 -*-
# @Time : 2020/11/6 18:51
# @File : test.py
# @Software: PyCharm
# @Platform: Win10
# @Author: 数据人阿多

from sshtunnel import SSHTunnelForwarder

server=SSHTunnelForwarder(('192.168.1.1', 2222),
                          ssh_password='123456*',
                          ssh_username='root',
                          remote_bind_address=('127.0.0.1', 3306))

server.start()
print('SSH connect success')
server.stop(force=True)

print('OK')

```

console:
```
SSH connect success
OK
^CException ignored in: <module 'threading' from '/home/data/anaconda3/envs/pytorch1.5/lib/python3.7/threading.py'>
Traceback (most recent call last):
  File "/home/data/anaconda3/envs/pytorch1.5/lib/python3.7/threading.py", line 1307, in _shutdown
    lock.acquire()
KeyboardInterrupt
```

thanks！

**Comment by duo66** [comment]
![issue](https://user-images.githubusercontent.com/48944742/98359769-b0809500-2063-11eb-8948-437419346b2f.png)

**Comment by iblancasa** [comment]
Same problem here :(

**Comment by duo66** [comment]
> Same problem here :(. Everything is working properly in 3.6

I am try on linux , but not work , thanks

**Comment by pahaz** [comment]
Could you please show sshtunnel logs with `sshtunnel.DEFAULT_LOGLEVEL = 1`?

**Comment by pahaz** [comment]
Ok, I've found a problem. It will be fixed in 0.3.0!
Currently, you can set `sshtunnel.SSH_TIMEOUT = 0.1` as workaround.

Sorry.

**Comment by pahaz** [comment]
`sshtunnel==0.3.0` is released! if you still have any problem feel free to reopen the issue or open a new one.

**Comment by duo66** [comment]
The problem has been solved，when upgrade to `sshtunnel==0.3.1`

Thanks
